backend/src/types/user.ts

```diff
@@ -45,3 +45,9 @@ export interface AppContext {
   gitHubId?: string
   currentUser: UserType
 }
+
+export interface LocalUser {
+  username: string
+  password: string
+  email: string
+}
```

Comment:
Why do we need a LocalUser object? Why not add username, email & password to usertype we already have?

Reply:
Didn't want to change the current UserType to match database because it would have broken the current functionality. Should I still do it?

Reply:
agreed to address this later.